src/firebase/firebase.ts

```diff
@@ -0,0 +1,34 @@
+import * as firebase from 'firebase/app';
+import "firebase/auth";
+import "firebase/firestore";
+import * as firebaseui from 'firebaseui';
+
+
+const firebaseAuthConfig = {
```

Comment:
This file should probably exists top-level as it only handles firebase initialization.